lib/src/components/board/board.dart

```diff
@@ -13,10 +13,11 @@ class BoardStatus {
   const BoardStatus(this.analogs, this.digitalInterrupts);
 
   factory BoardStatus.fromProto(common.BoardStatus pbBoardStatus) {
-    const boardStatus = BoardStatus(<String, int>{}, <String, int>{});
-    pbBoardStatus.analogs.forEach((key, value) => boardStatus.analogs[key] = value.value);
-    pbBoardStatus.digitalInterrupts.forEach((key, value) => boardStatus.digitalInterrupts[key] = (value.value.toInt()));
-    return boardStatus;
+    final Map<String, int> analogs = {};
+    final Map<String, int> digitalInterrupts = {};
+    pbBoardStatus.analogs.forEach((key, value) => analogs[key] = value.value);
+    pbBoardStatus.digitalInterrupts.forEach((key, value) => digitalInterrupts[key] = (value.value.toInt()));
```

Comment:
This is a little cleaner IMO. Thoughts? 
```suggestion
    final Map<String, int> analogs = pbBoardStatus.analogs.map((key, value) => MapEntry(key, value.value));
    final Map<String, int> digitalInterrupts = pbBoardStatus.digitalInterrupts.map((key, value) => MapEntry(key, value.value.toInt()));
```

Reply:
yeah I like that, thank you!